File: src/B_10814/B_10814.java
package B_10814;

import java.io.*;
import java.util.StringTokenizer;

public class B_10814 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;
        StringBuilder[] p = new StringBuilder[201];
        StringBuilder sb = new StringBuilder();
        int N = Integer.parseInt(br.readLine());
        for(int i=0; i<p.length; i++){
            p[i]=new StringBuilder();
        }
        for(int i = 0; i < N; i++) {
            st = new StringTokenizer(br.readLine()," ");
            int age = Integer.parseInt(st.nextToken());
            String name = st.nextToken();
            p[age].append(age).append(' ').append(name).append('\n');
        }
        for(StringBuilder val : p) {
            sb.append(val);
        }
        System.out.println(sb);
    }
}

File: src/B_10814/B_10814.kt
package B_10814

import java.io.*
import java.util.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    var st: StringTokenizer
    val p = arrayOfNulls<StringBuilder>(201)
    val sb = StringBuilder()
    val N = br.readLine().toInt()
    for (i in p.indices) {
        p[i] = StringBuilder()
    }
    for (i in 0 until N) {
        st = StringTokenizer(br.readLine(), " ")
        val age = st.nextToken().toInt()
        val name = st.nextToken()
        p[age]!!.append(age).append(' ').append(name).append('\n')
    }
    for (`val` in p) {
        sb.append(`val`)
    }
    println(sb)
}